acceptance/analyzer_test.go

```diff
@@ -532,13 +564,13 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 										h.WithFlags(append(
 											dockerSocketMount,
 											"--network", registryNetwork,
-											"--env", "CNB_REGISTRY_AUTH="+cacheAuthConfig,
+											"--env", "CNB_REGISTRY_AUTH="+fixtures.authRegAuthConfig,
```

Comment:
I think we should use here the second return value from line 140.

Reply:
Marking as solved following https://github.com/buildpacks/lifecycle/pull/668#discussion_r674152756.